task  SA
self.device  cuda
/home/lcur1101/ATCS_group3/marc_data/dataset_fr_train.json
Traceback (most recent call last):
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 223, in <module>
    acc = pipeline(LM_model, task, prompt_generator)
  File "/home/lcur1101/ATCS_group3/code/pipeline_file.py", line 106, in pipeline
    train_dataloader = create_dataloader(
  File "/home/lcur1101/ATCS_group3/code/dataloaders.py", line 94, in create_dataloader
    marc_dataset = MARCDataset(data_path, tokenizer)
TypeError: MARCDataset.__init__() takes from 1 to 2 positional arguments but 3 were given
srun: error: r36n2: task 0: Exited with exit code 1
